### DISCUSSION TITLE
Player Provider: Exposing as a web radio stream possible (Radio Broadcast)?

### DISCUSSION THREAD

**Discussion by mrueg** [music-assistant/support#3006] [Q&A] [answered]
I have a device that can listen to webradio streams (a link with an m3u file).

Is it possible for music assistant to expose such a file and use that as an audio sink?

This would allow me to use the generated URL and put it into the device.

**Comment by OzGav** [comment, marked as answer]
i believe this is a duplicate of this feature request https://github.com/orgs/music-assistant/discussions/2410

**Reply by mrueg** [reply]
Thanks, yes that's what I'd like to use as well! For some reason I was unable to find this one. Sorry for the noise!
